shared/checks/oval/installed_OS_is_rhel8.xml

```diff
@@ -38,5 +41,14 @@
     <linux:name>redhat-release</linux:name>
   </linux:rpminfo_object>
 
-
+  <linux:rpminfo_test check="all" check_existence="at_least_one_exists" comment="redhat-release-coreos is version 8" id="test_rhel8_coreos" version="1">
+    <linux:object object_ref="obj_rhel8_coreos" />
+    <linux:state state_ref="state_rhel8_coreos" />
+  </linux:rpminfo_test>
+  <linux:rpminfo_state id="state_rhel8_coreos" version="1">
+    <linux:version operation="pattern match">^[0-9\.]*8.*$</linux:version>
```

Comment:
For reference and documentation about the regex -- RHEL CoreOS versioning differs from upstream.

In RHEL, this would be something like 410.8-4, ref:
https://access.redhat.com/downloads/content/redhat-release-coreos/410.8-4.el8/x86_64/fd431d51/package

Whereas upstream releases follow a different scheme as identified at https://coreos.com/releases/.

(Thanks to @redhatrises for the doc references!)